scripts/kubestellar

```diff
@@ -50,63 +50,28 @@ function echoerr() {
 }
 
 
-function ensure_imw() {
-    while IFS=':' read -ra items; do
-        for i in "${items[@]}"; do
-            if [ "$i" == "" ] ; then
-                break
-            fi
-            if ! kubectl ws "$i" &> /dev/null ; then #
-                if ! kubectl ws create "$i" --enter &> /dev/null ; then
-                    echoerr 'unable to create "$i"!'
-                    break
-                fi
-            fi
-        done
-    done <<< "$1"
-    if ! kubectl get apibinding "edge.kubestellar.io" &> /dev/null; then
-    	kubectl kcp bind apiexport root:espw:edge.kubestellar.io
-    fi
-}
-
 function ensure_imws() {
     initial_ws=$(kubectl ws . --short)
     while IFS=',' read -ra items; do
         for i in "${items[@]}"; do
             if [ "$i" == "" ] ; then
                 continue
             fi
-            echo "Ensuring IMW: $i"
-            ensure_imw "$i"
-            kubectl ws use $initial_ws &> /dev/null # go back to the initial state
-        done
-    done <<< "$1"
-    kubectl ws use $initial_ws &> /dev/null
-}
-
-
-function ensure_wmw() {
-    while IFS=':' read -ra items; do
-        len=${#items[@]}
-        if [[ ${len} -eq 0 ]]; then
-            break # nothing to do
-        fi
-        # separate the last item
-        last_item=${items[$len-1]}
-        unset items[$len-1]
-        for i in "${items[@]}"; do
-            if [ "$i" == "" ] ; then
-                break
-            fi
-            if ! kubectl ws "$i" &> /dev/null ; then #
-                if ! kubectl ws create "$i" --enter &> /dev/null ; then
-                    echoerr 'unable to create "$i"!'
+            imw=${i#"root:"}
+            echo "Ensuring IMW: $imw"
+            kubectl ws root &> /dev/null
+            if ! kubectl ws "$imw" &> /dev/null ; then #
+                if ! kubectl ws create "$imw" --enter &> /dev/null ; then
+                    echoerr 'unable to create "$imw"!'
```

Comment:
It would be better to display the error message from `kubectl ws create` in this case.
Does not have to be addressed in this PR.